### ISSUE TITLE
[Feature Request]: Interactive Team Details Pages

### ISSUE THREAD

**Issue by kanCHI-Singhal** [mansiruhil/AmbuFlow#351]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Feature Description

This feature enables users to click on icons within the "Teams" section (e.g., "Service Manager," "Rescue Specialist") to view detailed information about each team member or group. When an icon is clicked, the website will display a dedicated page or popup with profiles, roles, and other relevant information.

### Use Case

Users visiting the site can click on team icons to access specific information about each team, such as their responsibilities, expertise, and team members. This provides a more engaging and informative experience, allowing users to better understand the services and personnel available.

### Benefits

1. Improved User Experience: Offers an interactive and engaging way for users to explore the teams and their roles.
2. Transparency: Provides clear information about the teams, building trust and credibility with users.
3 .Efficient Navigation: Allows users to access relevant information quickly and easily by clicking on icons, enhancing website functionality.
4 .Enhanced Engagement: Keeps users engaged by offering more in-depth information, encouraging them to explore the site further.

I hope may you understand the feature, if not then please tell me i will explain it more clearly.

Please assign me this issue under the labels of Gssoc-extd, hacktoberfest, hacktoberfest-accepted and relevant level.

### Priority

High

### Record

- [X] I agree to follow this project's Code of Conduct
- [X] I'm a GSSOC contributor
- [X] I want to work on this issue
- [X] I'm willing to provide further clarification or assistance if needed.

**Comment by kanCHI-Singhal** [comment]
Thank you for assigning me this issue. I will resolve this issue very soon .

**Comment by mansiruhil13** [comment]
> Thank you for assigning me this issue. I will resolve this issue very soon .

actually homepage broken, you have to work on this issue with that.

**Comment by kanCHI-Singhal** [comment]
Hey !, if you don't mind, Can you please assign level 2 to this issue . it will be very helpful for me.
Its my humble request . else there is no issue I will work over it with level 1 as well.

**Comment by mansiruhil13** [comment]
![Screenshot 2024-10-08 113439](https://github.com/user-attachments/assets/4814d338-cf43-46fa-9e30-65707d000620)

**Comment by kanCHI-Singhal** [comment]
Yes I understand , I will develop the complete teams page for this site., that when the user click on the teams page on the navbar , then he will get to see the team members and their details

**Comment by kanCHI-Singhal** [comment]
Thank you so much for assigning level 2 on this issue, I will resolve this very soon.

**Comment by kanCHI-Singhal** [comment]
Hey @mansiruhil13 ! I  had merge the PR , but I am getting conflicts in it. can you please help me to resolve them , Actually I am new to GitHub so I had not such knowledge that how to resolve them. It will be highly appreciable if you help me to resolve these conflicts.
